deploy.sh

```diff
@@ -23,4 +23,4 @@ sentry-cli deploys -o $SENTRY_ORG new -p $SENTRY_PROJECT -r $RELEASE -e $ENVIRON
 # This deploys React - The release was set in the static prod build
 gcloud app deploy --quiet
 
-cd ../flask && gcloud app deploy --quiet
+# cd ../flask && gcloud app deploy --quiet
```

Comment:
This is not a big deal at all, but I suspect this was included unintentionally so I wanted to point out you can reset your commits (`git reset` same as `git reset --mixed`) and then re-stage only the changes you want by using the interactive `git add -p` which is super neat. You can reset to previous commit with `HEAD^` or any commit and your files will remain unchanged (reset command will basically undo `git add -A` and `git commit`) so that you can selectively pick the individual chunks or changed lines to include in the commit.

so in this case I would reset all 3 commits and then interactively stage only the changes that I want like this. And then overwrite PR branch history with `--force`.
```
git reset 988da39d^
git add -p
...
git commit -m ...
git push -f
```

Reply:
Hmm I'm not sure I follow. I commented this out temporarily because I was deploying to Staging. I can uncomment it and commit that whenever I'm next editing code and committing, on this branch.

I guess the only one I need is the first one, I think.

I think I follow all what you're saying but I don't want to make a mistake. read through and noted all your commands offline.

I put the pg_sleep back because the engineer told me to.